modules/apps/web-experience/staging/staging-security/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-	provided group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "1.0.2"
+	provided group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
```

Comment:
@brianchandotcom Using "default" because we need to use 1.0.3 of portal-kernel. But I think only you can make this change. Please let me know if I'm wrong. Thanks.